CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# This codebase has shared ownership and responsibility.
+* @hashicorp/terraform-core
+* @hashicorp/terraform-devex
```

Comment:
I need to file a ticket with IT to get the devex team the write access to this repo, but will wait for this to be approved.

Reply:
Have access now, so this is good